src/game_events/action_wml.cpp

```diff
@@ -1684,7 +1684,10 @@ WML_HANDLER_FUNCTION(unstore_unit, /*event_info*/, cfg)
 			resources::units->erase(loc);
 			resources::units->add(loc, *u);
 
-			std::string text = cfg["text"];
+			config::attribute_value text = var["gender"].str() == "female" ? cfg["female_text"] : cfg["male_text"];
```

Comment:
Why do you use config::attribute_value and .blank() instead of just std::string and .empty()?